OracleTuxedo/core/samples/mp/Dockerfile

```diff
@@ -3,17 +3,20 @@
 # 
 
 # Pull base image
-FROM oracle/tuxedo:<VERSION>
+FROM oracle/tuxedo:12.2.2
 
-MAINTAINER Judy Liu<judy.liu@oracle.com>
-COPY simpapp_runme.sh simpappmp_runme.sh sleep.sh start_tlisten.sh /u01/oracle/
+MAINTAINER Todd Little<todd.little@oracle.com>
+COPY simpappmp_build.sh simpappmp_runme.sh sleep.sh start_tlisten.sh /u01/oracle/
 
 USER root
 RUN chown oracle:oracle -R /u01/oracle
 
 USER oracle
-RUN hostname; mkdir /u01/oracle/simpappmp
+RUN mkdir /u01/oracle/simpappmp
 
 WORKDIR /u01/oracle/simpappmp
 
-RUN sh -x ../simpappmp_runme.sh
+RUN sh -x ../simpappmp_build.sh
+
+# Define ENTRYPOINT. 
+ENTRYPOINT ["/u01/oracle/init.sh"]
```

Comment:
This will just run the upstream `init.sh` instead of running your sample. You need to create a script that actually runs your sample application, so it's useful.

Like the `shm` sample, you probably want to run `simpappmp_runme.sh` here.